client/web/src/enterprise/batches/detail/changesets/CreateCommentModal.tsx

```diff
@@ -0,0 +1,104 @@
+        setCommentBody(event.target.value)
+    }, [])
+
+    const onSubmit = useCallback<React.FormEventHandler>(
+        async event => {
+            event.preventDefault()
+            setIsLoading(true)
+            try {
+                const ids = await changesetIDs()
+                await createChangesetComments(batchChangeID, ids, commentBody)
+                afterCreate()
+            } catch (error) {
+                setIsLoading(asError(error))
+            }
+        },
+        [afterCreate, batchChangeID, changesetIDs, commentBody, createChangesetComments]
+    )
+
+    return (
+        <Dialog
+            className="modal-body modal-body--top-third p-4 rounded border"
+            onDismiss={onCancel}
+            aria-labelledby={LABEL_ID}
+        >
+            <div className="web-content">
+                <h3 id={LABEL_ID}>Post a bulk comment on changesets</h3>
+                <p className="mb-4">Use this feature to create a bulk comment on all the selected code hosts.</p>
+                {isErrorLike(isLoading) && <ErrorAlert error={isLoading} />}
+                <Form onSubmit={onSubmit}>
+                    <div className="form-group">
+                        <label htmlFor="token">Comment text</label>
+                        <textarea
+                            id="token"
+                            name="token"
+                            className="form-control text-monospace"
+                            placeholder={PLACEHOLDER_COMMENT}
+                            required={true}
+                            rows={8}
+                            minLength={1}
+                            value={commentBody}
+                            onChange={onChangeInput}
+                        />
+                    </div>
+                    <div className="d-flex justify-content-end">
+                        <button
+                            type="button"
+                            disabled={isLoading === true}
+                            className="btn btn-outline-secondary mr-2"
+                            onClick={onCancel}
+                        >
+                            Cancel
+                        </button>
+                        <button
+                            type="submit"
+                            disabled={isLoading === true || commentBody.length === 0}
+                            className="btn btn-primary"
+                        >
+                            {isLoading === true && <LoadingSpinner className="icon-inline" />}
+                            Post comments
+                        </button>
+                    </div>
+                </Form>
+            </div>
+        </Dialog>
+    )
+}
+
+const LABEL_ID = 'create-comment-modal-id'
+
+const PLACEHOLDER_COMMENT = `## Please review this
+
+This change is really important for us so please go review and merge this.`
```

Comment:
I think the placeholder should explain what's possible and provide a more general call to action, rather than being a specific example:

```suggestion
const PLACEHOLDER_COMMENT = `A comment that will be posted to all selected changesets.

You can use whatever formatting is available on your code host, such as _Markdown_!

Use this to request a review, provide an update, or post your favorite emoji, like 🦡.`
```